### PR TITLE
Adds ICefManager

### PR DESCRIPTION
**Pull request by ike709**
I'm getting OpenDream's unit testing working and we need to be able to not load CEF binaries.

### CODE REVIEW

File: Robust.Client.CEF/ICefManager.cs [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+{
+    public interface ICefManager
+    {
+        void Initialize();
+
+        void CheckInitialized();
+
+        void Update();
+
+        void Shutdown();
```
Comment (ike709) [inline Robust.Client.CEF/ICefManager.cs:11]:
```suggestion
        void Shutdown();
        
        IBrowserWindow CreateBrowserWindow(BrowserWindowCreateParameters createParams);
```

**Comment by ike709** [comment]
Checks will pass when someone updates content's RT version.

**Comment by PJB3005** [comment]
I feel like all APIs outside of the `CreateBrowserWindow` should be made internal. That does raise the question for how Robust.Client.CEF should be loaded though and I'm not sure how to go about that.

**Comment by PJB3005** [comment]
I guess the simplest approach would be that engine modules are declared in a resource manifest and loaded by the engine automatically. This does create some pains for loading it in dev environments though

**Comment by ike709** [comment]
...What do you actually want me to do here?

**Comment by ike709** [comment]
someone re-run checks

**Comment by ike709** [comment]
Semi-annual reminder that this is a thing.

**Comment by PJB3005** [comment]
Superseded by ba2f46424957d321db5504256824746bda0a3d9a

**Pull request closed without merging**